test/e2e/core/cluster_version.go

```diff
@@ -0,0 +1,40 @@
+package e2e
+
+import (
+	"fmt"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+
+	coreapi "k8s.io/api/core/v1"
+
+	util "github.com/openshift/cluster-node-tuning-operator/test/e2e/util"
+)
+
+// Test the core functionality by reporting host OS and cluster version.
+var _ = ginkgo.Describe("[core][cluster_version] Node Tuning Operator host OS and cluster version", func() {
+	var (
+		node *coreapi.Node
+	)
+
+	ginkgo.It("host OS and cluster version retrievable", func() {
+		ginkgo.By("getting a list of worker nodes")
+		nodes, err := util.GetNodesByRole(cs, "worker")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(len(nodes)).NotTo(gomega.BeZero(), "number of worker nodes is 0")
+
+		node = &nodes[0]
+		ginkgo.By(fmt.Sprintf("getting a Tuned Pod running on node %s", node.Name))
+		pod, err := util.GetTunedForNode(cs, node)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By(fmt.Sprintf("getting the host OS version on node %s", node.Name))
+		out, err := util.ExecCmdInPod(pod, "cat", "/host/etc/os-release")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		util.Logf("%s", out)
+
+		ginkgo.By("getting the cluster version")
+		_, _, err = util.ExecAndLogCommand("oc", "get", "clusterversion", "--no-headers")
```

Comment:
just a nit, why `--no-headers`? 
I think the command only logs this info and it's no parsed, so maybe better keeping the headers to understand what the booleans mean :)

```
NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.7.0     True        False         3h21m   Error while reconciling 4.7.0: the cluster operator ingress is degraded
```

Reply:
The reason I dropped it was that it was not aligned nicely. Perhaps devels are really only interested in the real clusterversion, such as 
```
oc get clusterversion -o jsonpath='{.status.desired.version}{"\n"}' version
```
would that make sense?

Reply:
indeed, makes sense

just FYI, I use this command for getting the version, but I don't know if it may give something different than yours:
```
oc version -o json | jq --raw-output '.openshiftVersion'
```
(unfortunately, `jq` is mandatory because `oc version` doesn't support `jsonpath` output :#)